src/React/DOM/Props.purs

```diff
@@ -894,8 +894,8 @@ onScrollCapture f = unsafeMkProps "onScrollCapture" (mkEffectFn1 f)
 onWheelCapture :: (SyntheticWheelEvent -> Effect Unit) -> Props
 onWheelCapture f = unsafeMkProps "onWheelCapture" (mkEffectFn1 f)
 
-ref :: (Nullable ReactRef -> Effect Unit) -> Props
-ref f = unsafeMkProps "ref" (mkEffectFn1 f)
+ref :: Ref.RefHandler HTMLElement -> Props
```

Comment:
Is this always `HTMLElement` here? For example, on React Native, would this be something different?

Reply:
Good question. I haven't used React Native myself, but I looked into this and as far as I can tell Native doesn't have refs - it seems to use something called [Direct Manipulation](https://facebook.github.io/react-native/docs/direct-manipulation) instead. This does raise the question of why the React team put refs in the React library instead of ReactDOM; I'm not sure what the answer is.